### PR TITLE
Introduce the Ukrainian language

### PR DESCRIPTION
**Pull request by JSMonk**

### CODE REVIEW

File: translation_uk.md
Comment (smorimoto) [inline translation_uk.md:1]: I'm now thinking that language-specific translation guides are unnecessary. The translator should be able to read English and copy-pasting in each language seems to be useless.

File: _data/languages.json [hunk 1]
```diff
@@ -19,6 +19,10 @@
     "title": "Русский",
     "id": "ru"
   },
+  {
+    "title": "Українська",
+    "id": "uk"
```
Comment (smorimoto) [inline _data/languages.json:24]: I'm not sure, but isn't it `uk-UA`?

```suggestion
    "id": "uk-UA"
```

Comment (chicoxyzzy) [reply _data/languages.json:24]: ISO 639-1 code is `uk` and country code `UA` shouldn't be necessary

**Review by chicoxyzzy — APPROVED** [review]
I am not a native speaker of Ukrainian, but based on my knowledge of the language, I can confirm that the translation seems correct.

**Review by smorimoto — APPROVED** [review]

**Pull request merged**